src/extras/geometries/BoxGeometry.js

```diff
@@ -124,4 +124,21 @@ THREE.BoxGeometry = function ( width, height, depth, widthSegments, heightSegmen
 THREE.BoxGeometry.prototype = Object.create( THREE.Geometry.prototype );
 THREE.BoxGeometry.prototype.constructor = THREE.BoxGeometry;
 
+THREE.BoxGeometry.prototype.clone = function () {
+
+	var geometry = new THREE.BoxGeometry(
+		this.parameters.width,
+		this.parameters.height,
+		this.parameters.depth,
+		this.parameters.widthSegments,
+		this.parameters.heightSegments,
+		this.parameters.depthSegments
+	);
+
+	THREE.Geometry.prototype.clone.call( this, geometry );
```

Comment:
It depends. Are modifications to the non-parametrized properties of a `BoxGeometry` allowed without flattening?

This fix is backward compatible with respect to code like this:

```
var pxGeometry = new THREE.PlaneGeometry( 100, 100 );
pxGeometry.faces[ 0 ].vertexColors = [ light, shadow, light ];
... 
copy = pxGeometry.clone(); // "exact" copy
```

Any tweaks to `.vertices`, `.faces` or `.faceVertexUvs` are copied. There are other properties like `.name` and `.colors` that seem relevant but are not currently handled by `Geometry.clone()`.

Should this use case be supported?